cognite/neat/_session/_set.py

```diff
@@ -23,14 +23,19 @@ def __init__(self, state: SessionState, verbose: bool) -> None:
         self._verbose = verbose
         self.instances = SetInstances(state, verbose)
 
-    def data_model_id(self, new_model_id: dm.DataModelId | tuple[str, str, str]) -> IssueList:
+    def data_model_id(self, new_model_id: dm.DataModelId | tuple[str, str, str], name: str | None = None) -> IssueList:
```

Comment:
Introduce this to avoid getting into this situation:
![image](https://github.com/user-attachments/assets/cf5d76d0-294e-455f-b924-b70ea004ff3a)